src/ResourceManagement/RecoveryServices.Backup/RecoveryServices.Backup.Tests/TestHelpers/BackupDefinition.cs

```diff
@@ -15,6 +15,12 @@
 
 namespace Microsoft.Azure.Management.RecoveryServices.Backup.Tests
 {
+    /// <summary>
+    /// For the recovery services backup tests, need some test artifact to be created manually for the tests to run:
```

Comment:
This is not sufficient.  You are using particular names for these resources, so you need to specify what they are, or provide a mechanism for a user to provide resource names and details without changing the code.